templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">
```

Comment:
The URL tag should use the correct namespace for the login view. Update it to `{% url 'blog:login' %}` to ensure it resolves correctly within the blog app.